### PR TITLE
Update linting

### PR DESCRIPTION
**Pull request by sam-github**

### CODE REVIEW

**Comment by sam-github** [comment]
PTAL

File: test/test-container-start-command.js [hunk 1]
```diff
@@ -6,14 +6,16 @@
...
-var basicOpts = {
-  instanceId: '_id',
-  baseDir: '_base',
-  server: {},
-};
+function newContainer() {
+  return new Container({
+    instanceId: '_id',
+    baseDir: '_base',
+    server: {},
```
Comment (pthieu) [inline test/test-container-start-command.js]: Will these parameters ever change? Not sure of deeper functionality so can't say if this needs an `options`.

**Comment by pthieu** [comment]
@sam-github: one question but LGTM

File: test/test-container-start-command.js (hunk 1 above)
Comment (sam-github) [inline test/test-container-start-command.js]: no, they are mandatory params that need some value in them, don't care what, just so we can have a Container and use it in the tests

**Pull request merged**